android/app/src/main/res/layout/fragment_add_experience.xml

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why frame layout and not linear layout? Frame layout should only have one child and it looks like you got 2 here